scarb/src/ops/fmt.rs

```diff
@@ -178,7 +213,7 @@ fn emit_formatted_file(
     match fmt.format_to_string(&path) {
         Ok(FormatOutcome::Identical(_)) => true,
         Ok(FormatOutcome::DiffFound(diff)) => {
-            target.emit(ws, path, &diff.formatted);
+            target.emit(ws, &diff.formatted);
```

Comment:
Don't we need the paths printed,  when formatting multiple files at once? How do we know which is which?

Reply:
when in emit mode the standard behavior from `forge` is to just concatenate each file without any info on the origin.   I do not know of any tool that uses the multiple files at once directly, generally they just do `tool fmt <filepath>` and expect to find the re-formatted file in stdout.

Reply:
also: even if the file is addentical, in emit mode, it should be emitted to stdout

Reply:
Hmm, can we still print paths in case there is multiple files under the path? 
The `--emit` has been implemented as a community request. Not sure if any tools rely on this, but this is how `cargo fmt -- --emit stdout` does it as well.

Reply:
Done - Now the behavior is like cargo's:

```
path/to/file.cairo:

<file content>
```

Reply:
FYI not emitting the path if formatting a single file only is okay to me :) Leaving the decision to you.

Reply:
ok, let's go with 2 then. It's easier to handle